hack/install_golangci.sh

```diff
@@ -12,6 +12,6 @@ if [ ! -x "$BIN" ]; then
     echo "Installing golangci-lint v$VERSION into $GOBIN"
     curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $GOBIN v$VERSION
 else
-    # Prints it's own file name as part of --verison output
+    # Prints it's own file name as part of --version output
```

Comment:
As long as you're touching this line, please remove the erroneous apostrophe in `it's`